common/core/desktop/src/mock/mock_processor.cpp

```diff
@@ -182,6 +182,9 @@ namespace km {
       return engine_attrs;
     }
 
+    km_kbp_keyboard_key  * mock_processor::get_key_list() const {
+      return nullptr;
```

Comment:
This returns nullptr but we don't define that as a possible output for the api function `km_kbp_get_key_list()` -- shouldn't we return an empty array instead?